README.md

```diff
@@ -1,3 +1,5 @@
 # Lucas Bootcamp
 
-Changes being made in the main branch, which will go to "PROD"
\ No newline at end of file
+Changes being made in the main branch, which will go to "PROD"
+
+Actually.... this is the actual fix....
```

Comment:
actually this is trash. Please fix

Reply:
hey actually I'm not sure we need this. DO you think we coudl do this in XYZ way instead?

Hey do you mind explaining this code block to me? I think its supposed to do Y, but it doesn't quite make sense yet